app.js

```diff
@@ -0,0 +1,282 @@
+// Global Variables to setup game
+let players = []
+let turn = 0  // 0 is X, 1 is O
+let gameMode = ''
+
+// Use arrays to record the moves that each player makes
+let xMoves = []
+let oMoves = []
+
+// get DOM object associated with start button, status area, timer and game mode (player v player or player v computer)
+let startButton = document.getElementById('start')
+let statusText = document.getElementById('status-text')
+let timer = document.getElementById('timer')
+let playerMode = document.getElementById('Player-vs-Player')
+let pcMode = document.getElementById('Player-vs-Comp')
+let xPlayerName = document.getElementById('x-player')
+let oPlayerName = document.getElementById('o-player')
+let newGame = document.getElementById('new-game')
+
+// get collection of DOM objects associated with each box on grid 
+let boxes = document.getElementsByClassName('cell')
+
+// setup listener on start button to detect click
+startButton.addEventListener('click', startGame)
+newGame.addEventListener('click', refreshGame)
+
+// setup strings that will be used to track where boxes where Xs and Os are placed
+let oMovesString = null;
+let xMovesString = null;
+
+// setup var for elapsed time
+let elapsedTime = null;
+
+
+function startGame() {
+    // set variable so we know game mode - i.e. player vs player or player vs computer
+    gameMode = document.querySelector('input[name=game-mode]:checked').value
+
+    // disable radio buttons and start button so player can't change during game
+    playerMode.disabled = true;
+    pcMode.disabled = true;
+    startButton.disabled = true
+    xPlayerName.disabled = true
+    oPlayerName.disabled = true
+
+    // set player names based on input
+    players[0] = xPlayerName.value
```

Comment:
This is where your games bugs out if you bypass the inputs, once you hit start, it disables the input areas, which then is caught in your clause on lines 60-67, you can fix this by setting the default value of a players name to the corresponding game token